install.sh

```diff
@@ -39,7 +39,8 @@ case "${ARCH}" in
 esac
 
 # This will allow things to work without sudo
-crew_folders="bin cache doc docbook etc include lib lib64 libexec man sbin share tmp var"
+crew_folders="bin cache doc docbook etc include lib libexec man sbin share tmp var"
+[ "${ARCH}" == "x86_64" ] && crew_folders="bin cache doc docbook etc include lib lib64 libexec man sbin share tmp var"
```

Comment:
How about `crew_folders="bin cache doc docbook etc include #{ARCH_LIB} libexec man sbin share tmp var"`?

Reply:
Just changed it to the bash version of that...